Library/Formula/ghc.rb

```diff
@@ -22,30 +21,26 @@ class Ghc < Formula
 
   if build.build_32_bit? || !MacOS.prefer_64_bit?
     resource "binary" do
-      url "https://www.haskell.org/ghc/dist/7.4.2/ghc-7.4.2-i386-apple-darwin.tar.bz2"
-      sha256 "80c946e6d66e46ca5d40755f3fbe3100e24c0f8036b850fd8767c4f9efd02bef"
+      url "https://downloads.haskell.org/~ghc/7.6.3/ghc-7.6.3-i386-apple-darwin.tar.bz2"
+      sha256 "cc298a03b5450e0cc93cf6929a47a31165733348d2d98f50627ea69532c19391"
```

Comment:
Don't bump these; we want to keep on an old version to support old versions of OSX.